scripts/Get-LatestGitHubRelease.ps1

```diff
@@ -0,0 +1,35 @@
+﻿function Get-LatestGitHubRelease {
+  <#
+      .SYNOPSIS
+          Gets the latest release of a given GitHub repository
+
+      .EXAMPLE
+          Get-LatestGitHubRelease cloudflare cloudflared
+  #>
+  [CmdletBinding()]
+  param(
+      # Repository owner
+      [Parameter(Mandatory, Position=0)]
+      [string]$Owner,
+
+      # Repository name
+      [Parameter(Mandatory, Position=1)]
+      [string]$Name,
+
+      # GitHub token, used to reduce rate-limiting or access private repositories (needs repo scope)
+      [string]$Token = "$($env:github_api_key)"
+  )
+  end {
+      $Request = @{
+          Uri = "https://api.github.com/repos/$Owner/$Name/releases/latest"
```

Comment:
Not so sure I like this being hard coded for the latest release only.
Any chance to make it allow a tag being specified by the user, but use `latest` by default?

Reply:
I would almost suggest that we have a separate function for getting a specific release tag - though at the moment, we don't have any use for that functionality in this repository (I think?). 

Would that be suited to a follow up PR / issue, where we have a use for it?

Reply:
@JPRuskin
FYI: The FreeCAD package uses the tag _weekly-build_